src/brod_group_coordinator.erl

```diff
@@ -286,6 +287,12 @@ commit_offsets(CoordinatorPid, Offsets0) ->
       {error, timeout}
   end.
 
+%% @doc Update the list of topics the brod_group_coordinator follow which
+%% triggers a join group rebalance
+-spec update_topics(pid(), [brod:topic()]) -> ok
+update_topics(CoordinatorPid, Topics) ->
+  CoordinatorPid ! {topics, Topics},
```

Comment:
style remark: perhaps change this to a gen_server:cast, and rename topic atom to `{add_topics` ?

Reply:
I was doing that at first, but it seems the rest of the group_coordinator already prefer `!`. No problem for changing the atom.